eng/common/pipelines/templates/jobs/perf.yml

```diff
@@ -113,6 +113,8 @@ jobs:
     parameters:
       AgentImage: $(OSVmImage)
       
+  - template: /eng/pipelines/templates/steps/install-dotnet.yml
```

Comment:
This isn't going to work in repos where this template doesn't exist. I think it only exists in the tools and .NET repo.